bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -21,6 +21,7 @@
   <![endif]-->
 
   <!--[if lt IE 9]>
+    {{ css_bundle('common-old-ie') }}
```

Comment:
Is this needed still?

Reply:
This page doesn't get the common IE bundle so without this the nav breaks in IE8.

Reply:
OK, so just to make sure I'm getting this right: We serve IE8 1st class page CSS, but the nav styling doesn't work well there any more, so we need to serve it the common-old-ie styles as well to compensate?